nucypher/blockchain/eth/sol/source/contracts/PolicyManager.sol

```diff
@@ -13,7 +13,7 @@ import "contracts/proxy/Upgradeable.sol";
 
 
 /**
-* @notice Contract holds policy data and locks fees
+* @notice Contract holds policy data and locks accrued policy fees until collection
```

Comment:
not only until collection, staker must fulfill a commitment before fee collection

Reply:
Can definitely remove 'until collection'. I put it there to preempt stakers reading the contract description and thinking their fees were locked for a similar duration as their stakes. Also, I don't think that 'until collection' implies that they _don't_ need to fulfil some obligations to get their fees, it's more neutral